linux/src/daemon_cl.cpp

```diff
@@ -429,6 +434,10 @@ int main(int argc, char **argv)
 			{
 				ether_phy_delay = iniParser.getPhyDelay();
 			}
+
+			portInit.allowNegativeCorrField = iniParser.getAllowNegativeCorrField();
+			GPTP_LOG_INFO("SyncFollowUp with negative correction field: %s",
+						  portInit.allowNegativeCorrField ? "permitted" : "forbiden");
```

Comment:
typo, should read "forbidden"

Reply:
It looks good to me. I would 'promote' the bail message to an ERROR and maybe direct the user to the config file to allow. Also, demote the WARNING to INFO. I want the 'user' to be able to suppress the message without suppressing  all warnings or modifying the code.